components/Footer.vue

```diff
@@ -39,8 +39,8 @@
         </div>
       </div>
       <hr class="border-white my-8">
-      <div class="flex justify-between items-center">
-        <div>
+      <div class="flex justify-between items-center flex-wrap">
+        <div class="hidden lg:block">
```

Comment:
I find it weird that we are hiding the get in touch CTA on mobile. I can see that the design indeed suggests that we remove. We are still able to access it from the hero section and the navigation items.

No action required here as long as we are aware that we are hiding this from the mobile views.